src/app/checkout/page.tsx

```diff
@@ -15,6 +15,11 @@ export default function CheckoutPage({ searchParams }: { searchParams: { checkou
 	return (
 		<div className="checkout-bg min-h-[calc(100vh-106px)]">
 			<section className="mx-auto max-w-7xl p-8">
+				<div className="flex items-center font-bold">
+					<a aria-label="homepage" href="/">
+						ACME
+					</a>
+				</div>
```

Comment:
Is this code not already present in the project?

Reply:
Oh that was my previous commit, didnt knew it got merged. Now how do I resolve this?

Reply:
@AryanPandeyy can you rebase?